### PR TITLE
Fix #399: Skip events test on www-dev.

### LINKED ISSUE #399: Skip testing major_link_destinations and major_link_urls for the home page, just on dev
From time to time (often enough) we fail to fetch the "Upcoming Events" feed from Mozilla Reps, on https://www-dev.allizom.org, which causes our tests on dev to fail.  Per IRC discussion, we should use the pytest.mark.skipif syntac (https://github.com/mozilla/Addon-Tests/commit/1663dd938f45e2ee842afa627962f9c4bc15856f#diff-147b19cd946fec500e242f83be6ccfb1L28) and skip running the couple of event/feed-related tests -- and just on dev -- per @pmclanahan (who also volunteered to help with a pull - thx ;-P)

https://bugzilla.mozilla.org/show_bug.cgi?id=1143814#c8

### PR DESCRIPTION
**Pull request by pmclanahan**

### CODE REVIEW

**Comment by pmclanahan** [comment]
@stephendonner r?

Couldn't use the skip decorator because this was really just one item in a list of things to test. So I'm optionally adding the item if it's not running against dev. Untested but should work. This may be hard to verify though.

**Comment by stephendonner** [comment]
I'm happy with the intent of the change, but it's a bit technical for me, so if @bobsilverberg or @davehunt could give this a look that would be great.

File: pages/desktop/home.py [hunk 1]
```diff
@@ -95,6 +88,17 @@ def go_to_page(self):
         },
     ]
 
+    def __init__(self, testsetup):
+        super(HomePage, self).__init__(testsetup)
+        if 'www-dev.allizom.org' not in self.base_url:
```
Comment (davehunt) [inline pages/desktop/home.py]: Short of a large refactor I think this is probably the most sensible approach. Could we add a comment to explain why we're doing this though? Also, it's not clear to me if this is temporary or something that will roll out across the environments - could we make that clear in the comment.

**Comment by pmclanahan** [comment]
Added a comment. @davehunt can you verify that this is what you wanted? Wasn't sure about the "across all environments" part.

**Comment by davehunt** [comment]
That's much clearer, thanks!

**Pull request merged**